libmscore/layout.cpp

```diff
@@ -2553,8 +2553,10 @@ void layoutDrumsetChord(Chord* c, const Drumset* drumset, const StaffType* st, q
 std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* tremolo, qreal stemLen1, qreal stemLen2)
       {
       const qreal spatium = tremolo->score()->spatium();
+#if 0
       const qreal sw = tremolo->score()->styleS(Sid::tremoloStrokeWidth).val();
       const qreal td = tremolo->score()->styleS(Sid::tremoloDistance).val();
+#endif
```

Comment:
How about moving these lines down to join the other disabled block of code?

Reply:
Good call. Will do.

Reply:
MinGW complains about those too BTW